draft-ietf-moq-warp.md

```diff
@@ -59,6 +59,14 @@ normative:
     title: "WebCodecs Codec Registry"
     date: September 2024
     target: https://www.w3.org/TR/webcodecs-codec-registry/
+  WEBVTT:
+    title: "World Wide Web Consortium (W3C), WebVTT: The Web Video Text Tracks Format"
+    date: April 2019
+    target: https://www.w3.org/TR/webvtt1/
+  IMSC1:
+    title: "W3C, TTML Profiles for Internet Media Subtitles and Captions 1.0 (IMSC1)"
+    date: April 2016
+    target: https://www.w3.org/TR/ttml-imsc1/
```

Comment:
i wonder of these needs to be informative references ?

Reply:
The idea was to provide explicit bindings for both those caption formats in order to drive interoperability, hence the normative references. I agree that the bindings are not in the spec and the only reference is the scope section,  so we could relax them to informative for now and then when the bindings are added in the future we can tighten to normative.